terraform/variables.tf

```diff
@@ -48,6 +49,12 @@ variable "gcp_sql_user_name" {
   description = "The username for the app component database"
   default     = "hexa-app"
 }
+variable "availability_type" {
```

Comment:
@mbayangdiop95 , a few comments:

- Please use the same naming convention as for the other variables (`gcp_sql_instance_...`)
- Let's move this variable a bit higher so that it is grouped with other `gcp_sql_instance_...` variables
- It also needs to be defined (and used ) in the `terraform_import` directory